api/scpca_portal/models/project.py

```diff
@@ -202,6 +202,18 @@ def get_download_config_file_output_name(self, download_config: Dict) -> str:
 
         return f"{file_name}.zip"
 
+    def get_computed_file(self, download_config: Dict[str, str | bool | int | None]):
```

Comment:
```suggestion
    def get_computed_file(self, download_config: Dict):
```

Reply:
Hmm, I thought this was taken care of in a previous commit. Sorry about that.